nemo_curator/classifiers/quality.py

```diff
@@ -36,108 +35,28 @@ class QualityModelConfig:
     max_len = 512
 
 
-# TODO: Remove this class after Quality Model is uploaded to HuggingFace
-class NCCustomModel(nn.Module):
-    def __init__(
-        self,
-        config: dataclass,
-        out_dim: int,
-        config_path: str = None,
-        pretrained: bool = False,
-        autocast: bool = False,
-    ):
-        super().__init__()
-        self.config = config
-        if config_path is None:
-            self.config = AutoConfig.from_pretrained(
-                config.model, output_hidden_states=True
-            )
-        else:
-            self.config = torch.load(config_path)
-
-        if pretrained:
-            self.model = AutoModel.from_pretrained(config.model, config=self.config)
-        else:
-            self.model = AutoModel(self.config)
-
-        self.fc_dropout = nn.Dropout(config.fc_dropout)
-        self.fc = nn.Linear(self.config.hidden_size, out_dim)
-        self._init_weights(self.fc)
-        self.autocast = autocast
-
-    def _init_weights(self, module):
-        if isinstance(module, nn.Linear):
-            module.weight.data.normal_(mean=0.0, std=self.config.initializer_range)
-            if module.bias is not None:
-                module.bias.data.zero_()
-        elif isinstance(module, nn.Embedding):
-            module.weight.data.normal_(mean=0.0, std=self.config.initializer_range)
-            if module.padding_idx is not None:
-                module.weight.data[module.padding_idx].zero_()
-        elif isinstance(module, nn.LayerNorm):
-            module.bias.data.zero_()
-            module.weight.data.fill_(1.0)
-
-    def feature(self, input_ids, attention_mask):
-        outputs = self.model(input_ids=input_ids, attention_mask=attention_mask)
-        last_hidden_states = outputs[0]
-        return last_hidden_states
-
-    def _forward(self, batch):
-        feature = self.feature(batch["input_ids"], batch["attention_mask"])
-        output = self.fc(self.fc_dropout(feature))
-        output = output.to(torch.float32)
-        return torch.softmax(output[:, 0, :], dim=1)
-
-    def forward(self, batch):
-        if self.autocast:
-            with torch.autocast(device_type="cuda"):
-                return self._forward(batch)
-        else:
-            return self._forward(batch)
-
-
 class QualityModel(HFModel):
-    def __init__(self, config, out_dim=None, model_path=None, autocast=False):
+    def __init__(self, config: dataclass, autocast: bool = False):
```

Comment:
Shoot I meant to do this in my PR, but can you change the type annotation here from `dataclass` to `QualityModelConfig`? Same with the domain classifier.